src/Kdyby/Console/DI/ConsoleExtension.php

```diff
@@ -82,18 +82,16 @@ public function loadConfiguration()
 		Nette\Utils\Validators::assert($config, 'array');
 		foreach ($config['commands'] as $i => $command) {
 			$def = $builder->addDefinition($this->prefix('command.' . $i));
-			list($def->factory) = Nette\DI\Compiler::filterArguments(array(
-				is_string($command) ? new Statement($command) : $command
-			));
-
-			if (class_exists($def->factory->entity)) {
-				$def->class = $def->factory->entity;
-			}
-
-			$def->setAutowired(FALSE);
-			$def->setInject(FALSE);
```

Comment:
Anyway, that should be out of the scope of this PR. We can continue this discussion in separate thread (if you still think that it would be good to change it). I would like to make more smaller steps with PR's.